src/sentry/discover/arithmetic.py

```diff
@@ -224,15 +223,14 @@ def parse_arithmetic(
     return result, list(visitor.fields)
 
 
-def resolve_equation_list(
-    equations: List[str], snuba_filter: eventstore.Filter
-) -> Dict[str, JsonQueryType]:
-    selected_columns = snuba_filter.selected_columns
+def resolve_equation_list(equations: List[str], selected_columns: List[str]) -> List[JsonQueryType]:
+    """Given a list of equation strings, resolve them to their equivalent snuba json query formats"""
+    resolved_equations = []
     for index, equation in enumerate(equations):
         # only supporting 1 operation for now
         parsed_equation, fields = parse_arithmetic(equation, max_operators=1)
         for field in fields:
-            if field not in fields:
+            if field not in selected_columns:
```

Comment:
🤡 I should've written a test for this earlier lol

Reply:
Yeah good idea let me backlog something for that I think eventually we could just treat any entries in arithmetic as selected_columns and handle them as needed
But for the short-term this makes equations a bit easier to work with so I'll leave it for now.